export_img.py

```diff
@@ -2,7 +2,7 @@
 Program export images in svg format to png format.
 Open file in directory where you have svg images saved.
 """
-
+import os
 import sys
```

Comment:
Používáš knihovnu `sys` někde?

Reply:
@encukou k čemu je knihovna sys? Přiznám se že jsem to jen zkopírovala jak jsi nám to předtím ukazoval, ale koukám že nepoužívám?
![image](https://user-images.githubusercontent.com/39279854/50157804-a23ff900-02d2-11e9-87ac-1737fe5901fc.png)

Reply:
Umí toho [spoustu](https://docs.python.org/3/library/sys.html), hlavně věci kolem samotného Pythonu. Všetšinou z ní člověk potřebuje něco konkrétního.
Nevím na co jsem ji používal já. Jestli ji nepoužíváš tak ji nepotřebuješ importovat.